docs/topics_5/rn-known-issues.adoc

```diff
@@ -12,9 +12,13 @@ At the time of release, the following known issues are identified as major issue
 |Component
 |Summary
 
+|link:https://issues.redhat.com/browse/WINDUP-2755[WINDUP-2755]
+|Eclipse Plugin
+|When running the MTA 5.0.1 plugin with Eclipse version 2020-06, analysis fails due to a null pointer exception.
```

Comment:
Sounds like a dangler. The analysis does not run the MTA plugin. :-) 

Suggested rewrite: "You cannot run the IDE plugin 5.0.1 with Eclipse 2020-06 because a null pointer exception causes analyses to fail. However, you can run the IDE plugin 5.0.1 with Red Hat CodeReady Studio 12.15."